src/questions/general-questions.md

```diff
@@ -34,3 +34,10 @@ permalink: /questions/general-questions/index.html
 * What role do you see yourself in?
 * Explain the difference between cookies, session storage, and local storage?
 * Can you explain what happens when you enter a URL into the browser?
+* Can you describe the components of a web application?
+* What is a database transaction and how does it work?
+* Describe the difference between SSR and CSR. Which one would you choose to render a web page?
```

Comment:
I would like a PR with just this question. I would change the wording of the follow up to be more of a discussion point. "which one would you choose" indicates that there's **one** answer where the answer is much mroe nuanced. It's great discussion, however. Please resubmit a PR with just this question